ci/resources.yml

```diff
@@ -2,6 +2,17 @@ resources:
   - name: dataworks-analytical-service-infra-pr
     type: pull-request
     source:
-      access_token: ((dataworks-secrets.concourse_github_pat))
+      access_token: ((github_com.token))
       repository: dwp/dataworks-analytical-service-infra
```

Comment:
this needs to be `repo` for the `jtarchie/pr` resource

Reply:
🤦